sdk-core/src/main/java/care/data4life/sdk/wrapper/URLEncoding.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.wrapper
+
+import java.net.URLDecoder
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets
+import java.util.Locale
+
+private fun Int.toHex(): String = Integer.toHexString(this).toUpperCase(Locale.US)
+
+object URLEncoding : WrapperContract.URLEncoding {
```

Comment:
I prefer instead of `URLEncoding` -> `UrlEncoding`

Reply:
seen in a lot of places

Reply:
done